scrunch/folders.py

```diff
@@ -127,12 +127,15 @@ def children(self):
         ds = self.root.dataset
         _children = []
         for item_url in self.folder_ent.graph:
-            if item_url in index:
-                subfolder = Folder(index[item_url].entity, self.root, self)
+            if item_url not in index:
+                continue
+            item_tup = index[item_url]
+            if item_tup['type'] == 'folder':
+                subfolder = Folder(item_tup.entity, self.root, self)
                 _children.append(subfolder)
             else:
                 # Add the variable
-                _children.append(ds[item_url])
+                _children.append(ds[item_tup['alias']])
```

Comment:
Could we make this `_children.append(item_tup['alias'])` instead so that iteration is faster? Users can then access only the variables they need to manipulate later.

Reply:
That would be too disruptive returning sometimes strings and sometimes Folder instances.